### PR TITLE
Add map to page

### PR DESCRIPTION
**Pull request by mffabrigas**
Use Google Maps API to display map

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,6 +27,17 @@ setInterval(() => {
...
 
+function start() {
+  loadComments();
+  createMap();
+}
+
+function createMap() {
+  const map = new google.maps.Map(
+    document.getElementById('map'), 
+    {center: {lat: 37.4220041, lng: -122.0862515}, zoom: 15});
```
Comment (chaowangzj) [inline portfolio/src/main/webapp/script.js]: Can you use constant var or comments to explain what this place is?

Comment (mffabrigas) [reply portfolio/src/main/webapp/script.js]: When you say constant var, do you mean make a descriptive constant i.e. MTV_LAT to hold the latitude and longitude values?

Comment (chaowangzj) [reply portfolio/src/main/webapp/script.js]: Yes, self documenting var name would be ideal.

Comment (mffabrigas) [reply portfolio/src/main/webapp/script.js]: Ok will do, thank you!

**Review by xiangchengz — APPROVED** [review]

**Pull request merged**